app/javascript/__tests__/dashboard.test.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env jest */
+'use strict'
+
+require('jest')
+var $ = require('jquery')
+var dt = require( 'datatables.net' )()
+var _ = require('lodash')
+global.$ = $
+//global.jQuery = $
+import { defineCaseContactsTable } from '../src/dashboard'
+
+describe('dashboard.js', () => {
+  describe('DataTable tests', () => {
+    function initializeDocumentWithTable (id, numberOfRows = 1, classes = '') {
+      var mockTable = `<table id=${id} class=${classes}>` +
+                      '<thead>' +
+                      '<tr> <th>Column1</th> <th>Column2</th> <th>Column3</th> </tr>' +
+                      '</thead> <tbody>'
+      _.range(numberOfRows).forEach((rowNum) => (
+        mockTable += '<tr> ' +
+                      `  <td>some data#${rowNum}</td>` +
+                      `  <td>other data#${rowNum}</td>` +
+                      `  <td>more data#${rowNum}</td> ` +
+                      '</tr>'
+      ))
+      mockTable += '</tbody> </table>'
+      document.body.innerHTML = mockTable
+    }
+
+    describe('tables with scroll enabled', () => {
+      beforeEach(() => {
+        initializeDocumentWithTable('case_contacts', 1, 'case-contacts-table')
+        defineCaseContactsTable()
+      })
+
+      test("case contacts table renders with 'width=100%'", () => {
+        /* This test ensures we retain the css workaround
+        *  to the "DataTables with scroll enabled result in
+        *  column/header misalignment" bug
+        *  (see https://datatables.net/manual/tech-notes/6)
+        */
+        $('table#case_contacts').each(function () {
+          expect($(this).css('width')).toEqual('100%')
```

Comment:
this is failing - maybe jest isn't loading the [stylesheet](https://github.com/rubyforgood/casa/blob/master/app/javascript/src/stylesheets/pages/case_contacts.scss)?

Reply:
running it locally, the test failed with
```
    Expected: "100%"
    Received: "0px"
```
but I think it's related to the [case_contacts stylesheet](https://github.com/rubyforgood/casa/blob/3575e4db62b4c55e94a9428a582822f4df028634/app/javascript/src/stylesheets/pages/case_contacts.scss#L32) not importing or something, because when i added a console.log to print out the table, some of the wrapper divs have `width: 100%`, but the tables have `width: 0px`:
```
<div class="dataTables_scrollHead" style="overflow: hidden; position: relative; border: 0px; **width: 100%**;">
  <div class="dataTables_scrollHeadInner" style="box-sizing: content-box; width: 0px; padding-left: 0px;">
      <table class="case-contacts-table dataTable no-footer" role="grid" style="margin-left: 0px; **width: 0px**;">
```

Reply:
based on [this comment](https://rubyforgood.slack.com/archives/CVB0QJGVD/p1602898489347600?thread_ts=1602891352.340000&cid=CVB0QJGVD) from the casa slack, it seems like i should switch testing gears from jest to Cypress. @compwron do you know of any Cypress examples in this repo? I haven't worked with Cypress before, and im worried that this seemingly small issue has snowballed & Im taking too long 🙃

Reply:
@gskifstad either jest or cypress are fine for now, we are fairly young to cypress, we only have the one example :) No worries, if you want to wrap it up you can mark the PR ready fr review :)